<?xml version="1.0" encoding="utf-8"?>
<assembly name="_/home/miki/Projects/goroot/src/xunit"
          run-date="2015-06-05" run-time="18:34:41"
          configFile="none"
          time="0.004"
          total="4"
          passed="3"
          failed="1"
          skipped="0"
          environment="n/a"
          test-framework="golang">

    <class time="0.004" name="_/home/miki/Projects/goroot/src/xunit"
  	     total="4"
  	     passed="3"
  	     failed="1"
  	     skipped="0">

        <test name="TestAdd"
          type="test"
          method="TestAdd"
          result="Pass"
          time="0.00">
        </test>

        <test name="TestSub"
          type="test"
          method="TestSub"
          result="Pass"
          time="0.00">
        </test>

        <test name="TestSubFail"
          type="test"
          method="TestSubFail"
          result="Fail"
          time="0.00">
          <failure exception-type="go.error">
             <message><![CDATA[	xunit_test.go:22: 3-1 != 3
		Some newline goes here]]></message>
      	  </failure>
      	</test>

        <test name="TestSubOK"
          type="test"
          method="TestSubOK"
          result="Pass"
          time="0.00">
        </test>

    </class>

</assembly>
